app/assets/stylesheets/components/_metadata-preview.scss

```diff
@@ -53,6 +53,16 @@ ul.sub-field {
   }
 }
 
+.two-column-fields-left {
```

Comment:
I believe these are a remnant from the preview gem changes I made for v1.14.  I don't think you used them, so I think they can be removed.

Reply:
removed